TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -38,6 +38,8 @@ def makeTreeFromMiniAOD(self,process):
         VarsInt                    = self.VarsInt,
         VarsBool                   = self.VarsBool,
         VectorTLorentzVector       = self.VectorTLorentzVector,
+        VectorXYZVector            = self.VectorXYZVector,
+        VectorXYZPoint             = self.VectorXYZPoint,
```

Comment:
for completeness the new branch lists should also be added here

Reply:
Done